docker-compose.yaml

```diff
@@ -105,7 +105,7 @@ services:
       ELASTICSEARCH_HOST: "${ELASTICSEARCH_HOST}"
       ELASTICSEARCH_PORT: "${ELASTICSEARCH_PORT}"
       ELASTICSEARCH_INDEX: "${ELASTICSEARCH_INDEX}"
-      LOG_PATH: "${LOG_PATH}"
+      LOG_PATH: "/var/log/automated_scripts/"
```

Comment:
Should this not stay ${LOG_PATH} for consistency ?

Reply:
purposely do that; otherwise wont work.